scripts/migration/renamings.json5

```
@@ -1447,5 +1447,9 @@
 
   'develop': [
     // New renamings go here!
+    {
+      oldName: 'Blockly.Mutator',
+      newName: 'Blockly.icons.MutatorIcon',
+    },
```

This is total pedantry because it won't actually affect the behaviour of the renaming script as presently written, but since the export is being renamed as well as the module:

```suggestion
    {
      oldName: 'Blockly.Mutator',
      newName: 'Blockly.icons.MutatorIcon',
      exports: {
        Mutator: {
          newExport: 'MutatorIcon',
          oldPath: 'Blockly.Mutator',
          newPath: 'Blocky.icons.MutatorIcon',
        },
      },
    },
```